packages/components/organisms/f-checkout/src/services/tests/addressService.test.js

```diff
@@ -174,4 +175,90 @@ describe('addressService', () => {
             });
         });
     });
+
+    describe('isAddressInLocalStorage ::', () => {
+        describe('if localStorage exists', () => {
```

Comment:
If we're checking when it exists do you want to check when it doesn't exist?

Reply:
Line 220 😉